src/main/java/de/hysky/skyblocker/utils/Utils.java

```diff
@@ -381,4 +394,21 @@ private static void resetLocRawInfo() {
         locationRaw = "";
         map = "";
     }
+
+    private static void initializeMayorCache() {
+        if (!mayor.isEmpty()) return;
+        CompletableFuture.supplyAsync(() -> {
+            try {
+                JsonObject json = JsonParser.parseString(Http.sendGetRequest("https://api.hypixel.net/v2/resources/skyblock/election")).getAsJsonObject();
+                if (json.get("success").getAsBoolean()) return json.get("mayor").getAsJsonObject().get("name").getAsString();
+                throw new IOException("API call for mayor status failed: " + json.get("cause").getAsString());
+            } catch (IOException | InterruptedException e) {
+                e.printStackTrace();
```

Comment:
Please use `LOGGER`. Also might as well catch all exceptions, or else the completable future fails silently.